### ISSUE TITLE
Display times in a city's local time? 

### ISSUE THREAD

**Issue by hampelm** [LocalData/sense-your-city#90]

**Comment by prashtx** [comment]
Can we display times in the browser's local time zone?

**Comment by hampelm** [comment]
looks like we can use [jstz](https://bitbucket.org/pellepim/jstimezonedetect)

**Comment by prashtx** [comment]
I think jstz is for timezone introspection, which we don't need, since the `Date` object is aware of the system's timezone offset (hence being able to get the date/time in local time, UTC time, or as an ISO string). Since moment wraps `Date`, you can pretty trivially put ISO strings in local time:
`moment('2015-01-30T20:00:00Z').format('D MMM [at] h:mm')` should yield `"30 Jan at 12:00"` in San Francisco. In fact `moment('2015-01-30T20:00:00Z').format()` will give you an equivalent ISO string but with the local timezone offset (`"2015-01-30T12:00:00-08:00"`).

The [moment docs](http://momentjs.com/docs/#/displaying/format/) have moment loaded on the page, making the console a nice moment REPL test area.

**Comment by hampelm** [comment]
I already use moment to format times for the graph axis -- `moment(value).format("hA")` gives times like `1PM`. 

I filed this bug because Shanghai light sensors peak around 7pm (browser local time), which feels a bit off to read (Shanghai is 13 hours ahead of me): 
![screen shot 2015-01-30 at 10 33 20 pm](https://cloud.githubusercontent.com/assets/86435/5986661/350af412-a8d0-11e4-9d5d-db9e46e764dc.png)

**Comment by prashtx** [comment]
Gotcha. If we show non-local time, can we indicate that it's "Shanghai time" somehow? I don't know that I'd expect that.

For that, I think jstz still wouldn't help, since we want the time zone of some other place. It might be easiest to record the appropriate IANA timezone for each city, and then we can use moment-timezone to do the math. When we expand beyond 7 cities, if we still want to show "sensor local time", we might have to use an external service.

**Comment by hampelm** [comment]
Agreed. If I get to it, I'll record the timezone in the city objects and use that to format w/moment. I'll note "sensor local time" or "Shanghai time" or similar somewhere noticeable
